### PR TITLE
Add globalkey to monorepo

### PR DESCRIPTION
**Pull request by willdoescode**

### CODE REVIEW

File: globalkey/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "globalkey",
```
Comment (benawad) [inline globalkey/package.json]: make this `@dogehouse/globalkey`

**Review by benawad — CHANGES_REQUESTED** [review]
also you probably want to add an action: `globalkey-npm_deploy.yaml` to publish to npm, you can copy/paste `dolma-npm_deploy.yaml`

**Comment by willdoescode** [comment]
> also you probably want to add an action: `globalkey-npm_deploy.yaml` to publish to npm, you can copy/paste `dolma-npm_deploy.yaml`

yeah Ill do that

**Comment by willdoescode** [comment]
done.

**Comment by willdoescode** [comment]
VERY NICE!

**Comment by amitojsingh366** [comment]
ill merge this now as i saw ben comment earlier and it seems he's cool with it

**Comment by willdoescode** [comment]
Swag

**Comment by cytoly** [comment]
swag

**Pull request merged**